### PR TITLE
Release v2.1.6 branch

### PR DESCRIPTION
**Pull request by keithboone**
Fixing an access control bug, and adding PHI masking to transactionData processErrorSummary and processErrorDetail

### CODE REVIEW

File: src/main/java/gov/cdc/izgateway/logging/event/Health.java [hunk 1]
```diff
@@ -85,7 +85,11 @@ public class Health {
 	@JsonProperty 
 	@Schema(description="Host name as known by the operating system")
 	private String hostname;
-
+	
+	@JsonProperty
+	@Schema(description="The database in use")
+	private String database;
```
Comment (pcahillai) [inline src/main/java/gov/cdc/izgateway/logging/event/Health.java:91]: Is this change related to dynamodb? Is it intended for this release?

**Review by pcahillai — APPROVED** [review]
I added a couple of questions.  Otherwise this looks good.

File: src/main/java/gov/cdc/izgateway/configuration/AppProperties.java [hunk 2]
```diff
@@ -21,6 +21,11 @@ public class AppProperties {
 	@Getter
 	@Value("${server.mode:prod}")
 	private String serverMode;
+	
+	@Getter
+	@Value("${spring.database:jpa}")
+	private String databaseType;
```
[the remark this answers is not in the document]
Comment (keithboone) [reply src/main/java/gov/cdc/izgateway/configuration/AppProperties.java:27]: Yes, it supports DynamoDb, but is harmless for 2.1.6

File: src/main/java/gov/cdc/izgateway/logging/event/Health.java (hunk 1 above)
Comment (keithboone) [reply src/main/java/gov/cdc/izgateway/logging/event/Health.java:91]: Yes, it supports DynamoDb, but is harmless for 2.1.6
